curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md

```diff
@@ -22,9 +22,9 @@ Newlines and other whitespace may be ignored unless contained within a quoted st
 
 Handling escaped quotes inside a string is optional; thus "`(foo"bar)`" may be treated as a string "`foo"bar`", or as an error.
 
-For this, the reader need not recognize "<code>\\</code>" for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
+For this, the reader need not recognize `\` for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
 
-Note that with the exception of "`()"`" ("<code>\\</code>" if escaping is supported) and whitespace there are no special characters. Anything else is allowed without quotes.
```

Comment:
I'm a little confused on this one - I'm wondering if these quotes should be added back? I can't figure out if they're supposed to be there or not.

Reply:
Alright, lets go with it then.